mmdet/apis/train.py

```diff
@@ -70,11 +71,23 @@ def _dist_train(model, dataset, cfg, validate=False):
     ]
     # put model on gpus
     model = MMDistributedDataParallel(model.cuda())
+
+    # fp16 setting
+    fp16 = cfg.get('fp16', None)
+    if fp16 is not None:
+        fp16_prepare_hook = Fp16PrepareHook(cfg.optimizer, **fp16.fp16_prepare)
```

Comment:
`**fp16.fp16_prepare` may not be useful, since BN always needs to be kept in the fp32 mode.